lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -192,24 +192,40 @@ nav {
   font-family: Helvetica, sans-serif;
   font-size: 14px;
   border-right: 1px solid #ccc;
-  position: sticky;
+  position: fixed;
   top: 0;
   overflow: auto;
+  z-index: 10;
 
   /* Layout */
-  width: 260px; /* fallback */
-  width: max(50px, 20vw);
-  min-width: 50px;
-  max-width: 80vw;
-  height: calc(100vh - 100px); /* reduce the footer height */
+  width: 300px;
+  min-height: 100vh;
+
   resize: horizontal;
+  background: white;
+}
+
+@media (min-width: 1024px) {
+  nav {
+    min-height: unset;
+    height: calc(100vh - 100px); /* reduce the footer height */
+  }
 }
 
 main {
   display: block;
-  margin: 1em;
+  margin: 3em 1em 1em;
   min-width: 340px;
   font-size: 16px;
+  width: 100%;
+  max-width: 64em;
```

Comment:
IMO, this could be even smaller.

Reply:
Can you explain why you think that's the case? (Asking as someone who has few css experience)

Reply:
Very wide lines make text harder to read. I think what I have here is enough, but we could go narrower if people want to.

Reply:
Here's a comparison on my monitor:

Max width of 1024px:
<img width="2560" alt="image" src="https://github.com/user-attachments/assets/6090abf8-edb8-456f-bf98-b734b50dc2d5">


Max width of 896px:
<img width="2560" alt="image" src="https://github.com/user-attachments/assets/26c8ccbf-7dae-418c-840d-a82b7f5b0ffd">

Reply:
Thanks for the explanation! Yeah I agree that the current full-width is not ideal. I don't have preference between 1024px and 896px tho, so let's keep it as is for now.